src/gui/accountsettings.cpp

```diff
@@ -196,17 +196,38 @@ AccountSettings::AccountSettings(const AccountStatePtr &accountState, QWidget *p
     connect(_accountState.data(), &AccountState::stateChanged, this, &AccountSettings::slotAccountStateChanged);
     slotAccountStateChanged();
 
+    connect(ui->addButton, &QPushButton::clicked, this, &AccountSettings::slotAddFolder);
+    connect(_accountState.data(), &AccountState::stateChanged, this, [this](AccountState::State state) {
+        ui->addButton->setEnabled(state == AccountState::Connected);
+
+        if (state == AccountState::Connected) {
+            if (_accountState->supportsSpaces()) {
+                ui->addButton->setToolTip(tr("Click this button to add a Space."));
```

Comment:
```suggestion
                ui->addButton->setToolTip(tr("Click this button to add a space."));
```

Reply:
Independent of that should space be translatable? If not we should remove it from the tr().
Else we might end up with forms of "Welträume"....

Reply:
My opinion is that we actually _should_ treat it as a proper noun, for that reason.

But if not used as a proper noun, don't leave it out of the translatable string. That would be very awkward for both developers as well as translators. I would go with a translation guide explaining the specific reason not to translate it.

Reply:
> Independent of that should space be translatable? If not we should remove it from the tr(). Else we might end up with forms of "Welträume"....

Let's go with "Galaxien". :-) Gut question, but for now I'd leave it translatable. I don't think the decision to keep "Space(s)" in German is final yet.

Reply:
> My opinion is that we actually _should_ treat it as a proper noun, for that reason.

We don't have one particular space but many. Take a look at Confluence, they don't use capitalization for their spaces either, not in regular sentences.